platform/android/java/lib/src/org/godotengine/godot/GodotLib.java

```diff
@@ -106,7 +106,7 @@ public class GodotLib {
 	/**
 	 * Forward double_tap events from the main thread to the GL thread.
 	 */
-	public static native void doubleTap(int buttonMask, int x, int y);
+	public static native void doubletap(int x, int y);
```

Comment:
Shouldn't this stay `doubleTap` in camelCase to match `double_tap` in snake case?

Reply:
I've changed it back to `doubleTap` again.